docs/monitoring/02-01-monitoring.md

```diff
@@ -2,32 +2,46 @@
+- **Alertmanager** that receives collected metrics from Prometheus and forwards this data onto configured Slack or Victor Ops channels.
 
-Pods may contain applications with custom metrics that Prometheus can poll through the Prometheus exporter.
+  > **NOTE:** There are no notification channels configured in the default monitoring installation. The current configuration allows you to add either Slack or Victor Ops channels.
 
-## Grafana
+  For more details, [read](https://prometheus.io/docs/alerting/alertmanager/) the official documentation.
 
-Kyma employs Grafana as a third-party resource in `kube-prometheus` to deliver a feature-rich metrics dashboard and graph editor.
+## Related resources
 
-To access the Grafana UI, use the following URL: `https://grafana.{DOMAIN}`. Replace DOMAIN with the domain of your Kyma cluster.
+Monitoring in Kyma also relies heavily on these custom resources:
 
-## Alertmanager
+- **Alert rules** define alert conditions for metrics. They are configured in Prometheus as custom resource definitions. Kyma provides a set of out-of-the-box alert rules that are passed from Prometheus to the Alertmanager. The definitions of such rules contain the alert logic, value at which alerts are triggered, alerts' severity, and more. If you pre-define specific Slack or Victor Ops channels, the Alertmanager fires the alerts onto the channel each time the alerts are triggered.
 
-Alertmanager receives harvested metrics from Prometheus and forwards this data on to the configured channels, such as email or incident management systems.
+- **Service Monitors** are custom resource definitions that specify the endpoints from which Prometheus should poll the metrics. Even if you expose a handful of metrics in your application, Prometheus polls only those from the endpoints specified in a Service Monitor custom resource definitions.
+
+## End-to-end monitoring flow
+
+The complete monitoring functionality and metric flow in Kyma comes down to these components and steps:
+
+![](./assets/monitoring-architecture.svg)
+
+1. Upon Kyma installation on a cluster, Prometheus Operator creates a Prometheus instance with the default configuration.
+2. The Prometheus server calls the endpoints of services specified in Service Monitors and collects all metrics exposed on those endpoints. Prometheus stores these metrics in a time-series database.
+3. If Prometheus detects any anomalies in metrics that are covered by the alert rules, Prometheus fires them and passes the information to the Alertmanager.
+4. If you manually configure a notification channel, you can instantly receive detailed information on the metric alerts detected by Prometheus.
+5. You can monitor metrics and track their historical data on Grafana dashboards.
```

Comment:
explore/vizualise instead of monitor.
Monitoring = metrics + alerts, in grafana you only watch